src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -134,6 +135,19 @@ tnsr::aa<DataType, SpatialDim, Frame> pi(
  * See Eqs. 8 and 9 of \cite Lindblom2005qh
  */
 template <size_t SpatialDim, typename Frame, typename DataType>
+void gauge_source(
+    gsl::not_null<tnsr::a<DataType, SpatialDim, Frame>*> gauge_source,
```

Comment:
note clang-tidy complains about `gauge_source` here vs `gauge_source_h` in the cpp